OneSignalSDK/onesignal/core/src/main/java/com/onesignal/core/internal/operations/impl/OperationRepo.kt

```diff
@@ -101,6 +103,13 @@ internal class OperationRepo(
         }
     }
 
+    fun <T : Any> containsParameterOfType(
```

Comment:
(ignore if this is just a test function we delete later) Shouldn't this be called `getParametersOfType` and return a list?